app/views/application/_navigation_main.html.slim

```diff
@@ -1,3 +1,6 @@
 ul.left
   - if user_signed_in?
     = active_link_to "Home", root_path, active: :exclusive, wrap_tag: :li
+    - policy = PagePolicy.new(current_user, nil)
```

Comment:
PagePolicy.new(current_user, nil)

you better not pass 'nil' here, it better to create 'default value' for page parameter in policy constructor